climakitae/threshold_tools.py

```diff
@@ -295,20 +366,40 @@ def get_return_value(
     conf_int_upper_bound=97.5,
     multiple_points=True,
 ):
-    """
+    """Creates xarray Dataset with return values and confidence intervals from maximum series
+
     Returns dataset with return values and confidence intervals from maximum series.
+
+    Parameters
+    ----------
+    ams: xarray.DataArray
+    return_period: float
+    distr: str
+    bootstrap_runs: int
+    conf_int_lower_bound: float
+    conf_int_upper_bound: float
+    multiple_points: boolean
+
+    Returns
+    -------
+    xarray.Dataset
     """
 
     data_variable = "return_value"
-    lmom_distr = get_lmom_distr(distr)
+    distr_func = get_distr_func(distr)
     ams_attributes = ams.attrs
 
     if multiple_points:
-        ams = ams.stack(allpoints=["y", "x"]).squeeze().groupby("allpoints")
+        ams = (
+            ams.stack(allpoints=["y", "x"])
+            .dropna(dim="allpoints")
```

Comment:
I know there was some back and forth about this on slack, but just want to make sure: is this `dropna` necessary? I'm wondering what happens during `unstack` later if we've dropped certain values after stacking, and if the behavior is still correct? (do we know why it is now needed but wasn't necessary in the previous code?)

Reply:
It is very necessary as the NaNs created by clipping result in error in PDF. That is a good question about the unstacking and we should test that outputs to be certain.